metrics.go

```diff
@@ -5,6 +5,17 @@ import (
 )
 
 var (
+	// Size buckets from 256 KiB (default chunk in Kubo) to 4MiB (maxBlockSize), 256 KiB  wide each
+	blockSizeHistogram = prometheus.LinearBuckets(262144, 262144, 16)
```

Comment:
:information_source: This gives us more useful buckets (same width), and we can plot them in Grafana using [heatmap widget](https://grafana.com/docs/grafana/v8.4/visualizations/heatmap/)